### ISSUE TITLE
Does not correctly recognize url(data:image/png:base64  

### ISSUE THREAD

**Issue by PeterLenahan** [CSSLint/csslint#442]
If you try and validate the css below, the tool things that there is a syntax error.

``` css
.leaflet-control-layers-toggle {
    background-image: url(data:image/png;base64,
iVBORw0KGgoAAAANSUhEUgAAABoAAAAaCAYAAACpSkzOAAAACXBIWXMAAAsTAAALEwEAmpwYAAAK
T2lDQ1BQaG90b3Nob3AgSUNDIHByb2ZpbGUAAHjanVNnVFPpFj333vRCS4iAlEtvUhUIIFJCi4AU
kSYqIQkQSoghodkVUcERRUUEG8igiAOOjoCMFVEsDIoK2AfkIaKOg6OIisr74Xuja9a89+bN/rXX
Pues852zzwfACAyWSDNRNYAMqUIeEeCDx8TG4eQuQIEKJHAAEAizZCFz/SMBAPh+PDwrIsAHvgAB
eNMLCADATZvAMByH/w/qQplcAYCEAcB0kThLCIAUAEB6jkKmAEBGAYCdmCZTAKAEAGDLY2LjAFAt
AGAnf+bTAICd+Jl7AQBblCEVAaCRACATZYhEAGg7AKzPVopFAFgwABRmS8Q5ANgtADBJV2ZIALC3
AMDOEAuyAAgMADBRiIUpAAR7AGDIIyN4AISZABRG8lc88SuuEOcqAAB4mbI8uSQ5RYFbCC1xB1dX
Lh4ozkkXKxQ2YQJhmkAuwnmZGTKBNA/g88wAAKCRFRHgg/P9eM4Ors7ONo62Dl8t6r8G/yJiYuP+
5c+rcEAAAOF0ftH+LC+zGoA7BoBt/qIl7gRoXgugdfeLZrIPQLUAoOnaV/Nw+H48PEWhkLnZ2eXk
5NhKxEJbYcpXff5nwl/AV/1s+X48/Pf14L7iJIEyXYFHBPjgwsz0TKUcz5IJhGLc5o9H/LcL//wd
0yLESWK5WCoU41EScY5EmozzMqUiiUKSKcUl0v9k4t8s+wM+3zUAsGo+AXuRLahdYwP2SycQWHTA
4vcAAPK7b8HUKAgDgGiD4c93/+8//UegJQCAZkmScQAAXkQkLlTKsz/HCAAARKCBKrBBG/TBGCzA
BhzBBdzBC/xgNoRCJMTCQhBCCmSAHHJgKayCQiiGzbAdKmAv1EAdNMBRaIaTcA4uwlW4Dj1wD/ph
CJ7BKLyBCQRByAgTYSHaiAFiilgjjggXmYX4IcFIBBKLJCDJiBRRIkuRNUgxUopUIFVIHfI9cgI5
h1xGupE7yAAygvyGvEcxlIGyUT3UDLVDuag3GoRGogvQZHQxmo8WoJvQcrQaPYw2oefQq2gP2o8+
Q8cwwOgYBzPEbDAuxsNCsTgsCZNjy7EirAyrxhqwVqwDu4n1Y8+xdwQSgUXACTYEd0IgYR5BSFhM
WE7YSKggHCQ0EdoJNwkDhFHCJyKTqEu0JroR+cQYYjIxh1hILCPWEo8TLxB7iEPENyQSiUMyJ7mQ
AkmxpFTSEtJG0m5SI+ksqZs0SBojk8naZGuyBzmULCAryIXkneTD5DPkG+Qh8lsKnWJAcaT4U+Io
UspqShnlEOU05QZlmDJBVaOaUt2ooVQRNY9aQq2htlKvUYeoEzR1mjnNgxZJS6WtopXTGmgXaPdp
r+h0uhHdlR5Ol9BX0svpR+iX6AP0dwwNhhWDx4hnKBmbGAcYZxl3GK+YTKYZ04sZx1QwNzHrmOeZ
D5lvVVgqtip8FZHKCpVKlSaVGyovVKmqpqreqgtV81XLVI+pXlN9rkZVM1PjqQnUlqtVqp1Q61Mb
U2epO6iHqmeob1Q/pH5Z/YkGWcNMw09DpFGgsV/jvMYgC2MZs3gsIWsNq4Z1gTXEJrHN2Xx2KruY
/R27iz2qqaE5QzNKM1ezUvOUZj8H45hx+Jx0TgnnKKeX836K3hTvKeIpG6Y0TLkxZVxrqpaXllir
SKtRq0frvTau7aedpr1Fu1n7gQ5Bx0onXCdHZ4/OBZ3nU9lT3acKpxZNPTr1ri6qa6UbobtEd79u
p+6Ynr5egJ5Mb6feeb3n+hx9L/1U/W36p/VHDFgGswwkBtsMzhg8xTVxbzwdL8fb8VFDXcNAQ6Vh
lWGX4YSRudE8o9VGjUYPjGnGXOMk423GbcajJgYmISZLTepN7ppSTbmmKaY7TDtMx83MzaLN1pk1
mz0x1zLnm+eb15vft2BaeFostqi2uGVJsuRaplnutrxuhVo5WaVYVVpds0atna0l1rutu6cRp7lO
k06rntZnw7Dxtsm2qbcZsOXYBtuutm22fWFnYhdnt8Wuw+6TvZN9un2N/T0HDYfZDqsdWh1+c7Ry
FDpWOt6azpzuP33F9JbpL2dYzxDP2DPjthPLKcRpnVOb00dnF2e5c4PziIuJS4LLLpc+Lpsbxt3I
veRKdPVxXeF60vWdm7Obwu2o26/uNu5p7ofcn8w0nymeWTNz0MPIQ+BR5dE/C5+VMGvfrH5PQ0+B
Z7XnIy9jL5FXrdewt6V3qvdh7xc+9j5yn+M+4zw33jLeWV/MN8C3yLfLT8Nvnl+F30N/I/9k/3r/
0QCngCUBZwOJgUGBWwL7+Hp8Ib+OPzrbZfay2e1BjKC5QRVBj4KtguXBrSFoyOyQrSH355jOkc5p
DoVQfujW0Adh5mGLw34MJ4WHhVeGP45wiFga0TGXNXfR3ENz30T6RJZE3ptnMU85ry1KNSo+qi5q
PNo3ujS6P8YuZlnM1VidWElsSxw5LiquNm5svt/87fOH4p3iC+N7F5gvyF1weaHOwvSFpxapLhIs
OpZATIhOOJTwQRAqqBaMJfITdyWOCnnCHcJnIi/RNtGI2ENcKh5O8kgqTXqS7JG8NXkkxTOlLOW5
hCepkLxMDUzdmzqeFpp2IG0yPTq9MYOSkZBxQqohTZO2Z+pn5mZ2y6xlhbL+xW6Lty8elQfJa7OQ
rAVZLQq2QqboVFoo1yoHsmdlV2a/zYnKOZarnivN7cyzytuQN5zvn//tEsIS4ZK2pYZLVy0dWOa9
rGo5sjxxedsK4xUFK4ZWBqw8uIq2Km3VT6vtV5eufr0mek1rgV7ByoLBtQFr6wtVCuWFfevc1+1d
T1gvWd+1YfqGnRs+FYmKrhTbF5cVf9go3HjlG4dvyr+Z3JS0qavEuWTPZtJm6ebeLZ5bDpaql+aX
Dm4N2dq0Dd9WtO319kXbL5fNKNu7g7ZDuaO/PLi8ZafJzs07P1SkVPRU+lQ27tLdtWHX+G7R7ht7
vPY07NXbW7z3/T7JvttVAVVN1WbVZftJ+7P3P66Jqun4lvttXa1ObXHtxwPSA/0HIw6217nU1R3S
PVRSj9Yr60cOxx++/p3vdy0NNg1VjZzG4iNwRHnk6fcJ3/ceDTradox7rOEH0x92HWcdL2pCmvKa
RptTmvtbYlu6T8w+0dbq3nr8R9sfD5w0PFl5SvNUyWna6YLTk2fyz4ydlZ19fi753GDborZ752PO
32oPb++6EHTh0kX/i+c7vDvOXPK4dPKy2+UTV7hXmq86X23qdOo8/pPTT8e7nLuarrlca7nuer21
e2b36RueN87d9L158Rb/1tWeOT3dvfN6b/fF9/XfFt1+cif9zsu72Xcn7q28T7xf9EDtQdlD3YfV
P1v+3Njv3H9qwHeg89HcR/cGhYPP/pH1jw9DBY+Zj8uGDYbrnjg+OTniP3L96fynQ89kzyaeF/6i
/suuFxYvfvjV69fO0ZjRoZfyl5O/bXyl/erA6xmv28bCxh6+yXgzMV70VvvtwXfcdx3vo98PT+R8
IH8o/2j5sfVT0Kf7kxmTk/8EA5jz/GMzLdsAAAAgY0hSTQAAeiUAAICDAAD5/wAAgOkAAHUwAADq
YAAAOpgAABdvkl/FRgAABJRJREFUeNqklk9IG1kcx78zk8mMhGRiYqKxJsE0CKVsWXtYFoqnHrSH
Wo899FLoSRYs6KE3ocUiJaFbkP4BqR4UqrCw6UFtaU+lKCsoeLGENJI/mubfmElIM5P5t5c1u7Mp
aay/23uPeZ/33nz4vkfouo526vr16+jt7cWNGzdmASASidw7OjpCtVpFO3MQ7YLevn17lWGYVw6H
owsAeJ4vSJJ0c3Fx8X2hUEC9Xm8J/C5obW3NQdP0a47jrvh8PlitVgBApVJBMpmEIAgfZVkeff78
OV8ul6EoyulBa2trsyzLTnq9XpPT6YTZbDaM1+t1FItFpFIpRRTF8PLy8r1sNouvX7827e6boEgk
cpWm6Vdut7vL4/Ggo6MDBEF8czG6rqNWqyGTySCXyxVkWb45Pz//nud5KIrSABpAKysr3Waz+Q+O
46709fXBYrHAZDI17QJA0+4URUG1WkU6nYYgCB9VVR2dm5vjq9UqVFX9F7S6uhpiGGbi3LlzJofD
AZqmDROpqgpRFEFRVKPNsmyjfVKyLIPneRweHiqSJD1ZWFiYKhaLIF68eDFCUdSSz+dzejwesCxr
OCZd1xsKMwxjAEmSBIIgYLFYmr4RRRGZTAbJZLKoquotkyRJdLlcJkVRBAD09PQ0JqvVahBFESzL
wmQyQVEUg1UEQUBRFGSzWbAsi46OjsYivnz5gmg0ilKpRNpsNrpxdI8ePfq9VCr9FggEKK/XC5Ik
QdM0WJYFSZItZdA0DaIoQpZlaJqGVCqFeDyu2u32uYWFhbuyLBtlePny5U+pVGqb53nmwoUL6O/v
b6zypDRNAwCQJGnor9VqODg4wP7+PhwOh9Tb2/vzzMzMJ03ToOv6t2Wo1+vY2NiArusIBoPo6uoC
SZJQVbUB0DQNFEVB0zQUCgXEYjEQBIGRkRGYzeaGDBMTE1MAWsuws7ODDx8+oLOzE16vF52dnQbQ
8fExUqkUjo+PMTQ0hMuXL/+YDIFAAE6nE1tbW9je3obf74fH4wEAZDIZJBIJ+P1+jI2NgeM4SJJ0
dhny+TzevHkDQRAAABzHYXh4GC6Xq6UMDx8+vNuUDO3IsLe3BwC4dOnSd2WYnJz81JR135Ph/1H0
3+hpJcOdO3fal8HlcsHv98NqtTbGdF1HpVJBIpFAPp8/mwwXL15EMBjEu3fvsLm5CZ/PZ5AhmUwi
EAjg9u3bYBjm9DKcP38edrvdkAbZbBbr6+solUoAALvdjmvXrqG7u9uQEqVSCZ8/f24tw+PHj7sF
QXitquovAwMD8Pv9YFnW8E92d3cBAIODg4Z+URSRSCQQjUZBUdRfHMeNTk9PZ1tefKFQaCSfzy9Z
rVbnwMAA3G53SxlyuRyi0SgqlUrR5XLdun///saprvIHDx6EisXihM/nMwUCAdhsNoMM5XIZ8Xgc
yWRScTqdT549ezb1Q28GAJidne0WBOFPSZJ+DQaD6OvrAwCk02nEYjEwDLPFcdxYOBzOngTumZ5b
MzMzI7lcbknTNOc/6V10u923wuHwRivAqUE2mw0URWF8fDwEAE+fPp1SVRXt1t8DAKzuGCketP3p
AAAAAElFTkSuQmCC);
    width: 36px;
    height: 36px;
    }
```
